processor/k8sprocessor/kube/fake_informer.go

```diff
@@ -88,3 +113,48 @@ func (c *FakeController) LastSyncResourceVersion() string {
 func (f *FakeInformer) SetWatchErrorHandler(cache.WatchErrorHandler) error {
 	return nil
 }
+
+type NoOpInformer struct {
+	*NoOpController
+}
+
+func NewNoOpInformer(
+	_ kubernetes.Interface,
+) cache.SharedInformer {
+	return &NoOpInformer{
+		NoOpController: &NoOpController{},
+	}
+}
+
+func (f *NoOpInformer) AddEventHandler(handler cache.ResourceEventHandler) {}
+
+func (f *NoOpInformer) AddEventHandlerWithResyncPeriod(handler cache.ResourceEventHandler, period time.Duration) {
+}
+
+func (f *NoOpInformer) GetStore() cache.Store {
+	return cache.NewStore(func(obj interface{}) (string, error) { return "", nil })
+}
+
+func (f *NoOpInformer) GetController() cache.Controller {
+	return f.NoOpController
+}
+
+type NoOpController struct{}
+
+func (c *NoOpController) HasSynced() bool {
+	return true
+}
+
+func (c *NoOpController) Run(stopCh <-chan struct{}) {}
+
+func (c *NoOpController) HasStopped() bool {
+	return true
+}
```

Comment:
```suggestion
type NoOpController struct{
  hasStopped bool
}

func (c *NoOpController) Run(stopCh <-chan struct{}) {
    go func() {
        <- stopCh
        c.hasStopped = true
    }()
}

 func (c *NoOpController) HasStopped() bool {
     return c.hasStopped
 }

 func (c *NoOpController) HasSynced() bool {
     return true
 }
```

Reply:
May be we should do this for correctness?

Reply:
Just in case something in future relies on HasStopped and does something accordingly.